### PR TITLE
Update create proposal flow

### PR DESCRIPTION
**Pull request by neokry**
## Description

simplify proposal creation by moving transaction queue to a modal

## Motivation & context

- users were getting stuck on the proposal creation page 
- we need more room to add previews for bigger proposal templates like droposals

## Code review

- are proposal template forms are still functioning correctly
- is the transaction queue modal retaining all functionality
## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [x] I have done a self-review of my own code
- [x] Any new and existing tests pass locally with my changes
- [x] My changes generate no new warnings (lint warnings, console warnings, etc)

### CODE REVIEW

**Review by isaaczaak — APPROVED** [review]
tested and approved

**Review by nadiakiamilev — APPROVED** [review]
nice changes!

File: apps/web/src/modules/proposal/components/ProposalNavigation.tsx [hunk 1]
```diff
@@ -66,7 +75,25 @@ export const ProposalNavigation: React.FC<ProposalNavigationProps> = ({ handleBa
...
+                enabled={!!transactions.length}
+                href={`/dao/${token}/proposal/review`}
+              >
+                <Button disabled={!transactions.length}>Continue</Button>
+              </OptionalLink>
+            </Flex>
+          </Box>
+        ) : null}
       </Flex>
+      <AnimatedModal close={() => setQueueModalOpen(false)} open={queueModalOpen}>
```
Comment (nadiakiamilev) [inline apps/web/src/modules/proposal/components/ProposalNavigation.tsx:94]: non-blocking comment and a ux question (so feel free not to address) - maybe we could consider adding a 'x' close button on the animated modal itself so then there wouldn't need to be an explicit close button within the modal content?

Comment (neokry) [reply apps/web/src/modules/proposal/components/ProposalNavigation.tsx:94]: good point will discuss with @isaaczaak

Comment (isaaczaak) [reply apps/web/src/modules/proposal/components/ProposalNavigation.tsx:94]: thanks @nadiakiamilev for the suggestion! we've opened an issue for this #196

**Pull request merged**